lib/courses/widgets/lesson_screen.dart

```diff
@@ -0,0 +1,89 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:schulcloud/app/widgets.dart';
```

Comment:
Wanted: Newline. Also, the `webview_flutter` import could be moved to the top right after the flutter package import